src/components/play-button/ASPlayButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { Button, Image, Modal, TouchableOpacity, Text } from 'react-native'
+import WebView from 'react-native-webview'
+
+import { playButtonIcon } from '../../constants/common-constants'
+
+import { styles } from './asPlayButton-styles'
+
+const ASPlayButton = () => {
+  const [modalVisible, setModalVisible] = useState(false)
```

Comment:
mention the type as well for example 
const [modalVisible, setModalVisible] = useState<boolean>(false)